ipa-core/src/query/processor.rs

```diff
@@ -468,11 +486,11 @@ mod tests {
     }
 
     fn helper_respond_ok() -> Arc<dyn RequestHandler<HelperIdentity>> {
-        prepare_query_handler(|_| async { Ok(HelperResponse::ok()) })
+        make_owned_handler(move |_req, _| futures::future::ok(HelperResponse::ok()))
     }
 
     fn shard_respond_ok(_si: ShardIndex) -> Arc<dyn RequestHandler<ShardIndex>> {
-        prepare_query_handler(|_| async { Ok(HelperResponse::ok()) })
+        make_owned_handler(move |_req, _| futures::future::ok(HelperResponse::ok()))
```

Comment:
we will have git conflicts here with the query_status API...
what approach do you want to keep?

Reply:
this one makes more sense to me as OK response for any request, not just prepare_query

Reply:
I mean this lines on my PR:

https://github.com/private-attribution/ipa/pull/1439/files#diff-49e05834cf3a03f064b79804a2c10dd7fa9606b61835ae9f71192e06cd6a8935R534-R548